libs/phpseclib/vendor/phpseclib/phpseclib/phpseclib/Crypt/RSA/Formats/Keys/PuTTY.php

```
@@ -56,7 +56,10 @@ public static function load($key, $password = '')
         if (!isset($components['private'])) {
             return $components;
         }
-        extract($components);
+        $type = $components['type'];
```

PHPMD (2.15.0)

Avoid unused local variables such as '$type'. (Unused Code Rules)

(UnusedLocalVariable)